appinventor/appengine/war/index.html

```diff
@@ -28,6 +28,10 @@
       </ul>
     </div>
     <script language="javascript" src="ode/ode.nocache.js"></script>
+    <script type="text/javascript" src="tutorials/tutorial.js"></script>
```

Comment:
In a word no. ode.nocache.js is generated by GWT and tutorial.js is not. It would be too painful and likely GWT version dependent to attempt to merge them. Also, tutorial.js can probably be cached (or not), but ode.nocache.js must not be cached. We take pains in the App Engine configuration to make this so.